src/BenchmarkDotNet.Core/BenchmarkDotNet.Core.csproj

```diff
@@ -23,7 +23,7 @@
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.1' Or '$(TargetFramework)' == 'netcoreapp1.1' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.1' Or '$(TargetFramework)' == 'netcoreapp2.0' ">
```

Comment:
good catch! I have no idea how did it work without it ;)

Reply:
@adamsitnik, does TargetFrameworkIdentifier in condition can simplify clause, example
https://github.com/aspnet/MusicStore/blob/dev/samples/MusicStore/MusicStore.csproj#L20,L24
what do you think?

Reply:
It makes much more sense to me. I think it's a good idea to use `TargetFrameworkIdentifier` everywhere (not only for `.NETCoreApp`, but also for `.NETFramework`).

Reply:
@Ky7m very good idea!

Reply:
Great, let me grab this item and I'll send a PR